### ISSUE TITLE
Adding edited works to author profile

### ISSUE THREAD

**Issue by B-Carlo** [WDscholia/scholia#2415]
I apologize if this topic is the same as https://github.com/WDscholia/scholia/issues/1259 and 1260, but I did not succeed in reading the content of the issue, and, as far as I understand, a issue with editor role is still present in Scholia.
In fact, I noted that works edited by an author are not shown in the "List of publications" by that author in his/her author profile. As for in Social sciences and Humanities editing the work of a classic author, or e.g. editing a miscellaneous volume about a defined topic choosing and inviting selected scholars, are meaningful intellectual works and are relevant for the evaluation of an author profile, I guess if it could be possible to show items with https://www.wikidata.org/wiki/Property:P98 towards an author in that author profile.
I hope I made my point sufficiently clear, and that this was the proper place and way to make it.

**Comment by fnielsen** [comment]
Yes, that could be a good idea. I wonder how we best can present the information without confusing author and editor information?

**Comment by B-Carlo** [comment]
I would suggest to represent editorial role in the same way used for roles
such as "first author", "solo author", "middle author" etc. in "Numbers of
publications per year" graph. Could it be a viable solution?
I guess that editorial roles should be added everywhere as all the
'authored' works of the profile, to give a complete image of the researcher.


Il giorno ven 2 feb 2024 alle ore 22:58 Finn Årup Nielsen <
***@***.***> ha scritto:

**Comment by fnielsen** [comment]
For the [List of publications](https://scholia.toolforge.org/author/Q20980928#list-of-publications) we would need to make a new column. In [Number of publications per year](https://scholia.toolforge.org/author/Q20980928#publications-per-year) we can also add it.

**Comment by B-Carlo** [comment]
Instead of creating a new column, which I guess would be too much complicated, why not just adding an >(ed.)< after the name of the person, in required cases? In fact, excluding completely such intellectual works from the list reduces quite a bit also the number of total publications. I did not mentioned yet translations: a translation from Latin or Greek into a modern language, or just a translation of Walt Whitman into, let's say, Italian or French or Spanish, can be a challenge of creativity as well as writing totally new poems. I work in a Humanities Department, and this gap in Scholia is highly relevant from my institutional point of view and prevent a lot my attempts to put my colleagues' attention on Wikidata as research hub. So far, STEM researchers' papers are usually evaluated by bibliometrics, but bibliometric assessment is becoming more and more relevant in Soc Sc. and Humanities. Another way to give evidence of this kind of intellectual works could be a complete duplication of "List of publication" , adding what could be named "list of editorial curatorships and translations" (I guess this title do not sound too much English). This could be interesting; in STEM this section could be completely ignored as not relevant, in the other fields would became a very interesting section to be read.

**Comment by fnielsen** [comment]
I have added the (ed.) to Synia here: https://synia.toolforge.org/#author/Q18618629 This seems to work ok.

**Comment by B-Carlo** [comment]
Yes, Finn, it does! Great! If possible, could the addition be done to Scholia too? Thanks a lot, this is a great improvement! 
Are translations going to be added too, in the same form? 
I did not know Synia; which is the difference with Scholia? Is it a sort of sandbox for Scholia?

**Comment by fnielsen** [comment]
I suppose we could also add translations.

For Synia the templates are defined on a wiki so users can edit them, - instead of just developers. For the author page the template page is https://www.wikidata.org/wiki/Wikidata:Synia:author
